packages/flutter_tools/templates/widget_preview_scaffold/lib/src/widget_preview.dart.tmpl

```diff
@@ -31,15 +31,33 @@ class WidgetPreview {
   const WidgetPreview({
     required this.builder,
     required this.scriptUri,
+    required this.line,
+    required this.column,
     required this.previewData,
     required this.packageName,
   });
 
+  @visibleForTesting
+  const WidgetPreview.test({
+    required this.builder,
+    required this.previewData,
+    this.scriptUri = '',
+    this.line = -1,
+    this.column = -1,
```

Comment:
Any reason we can't use `null` instead? We've had bugs in server/LSP before because it wasn't obvious from the types that we might get a -1 and they were used in bad maths - whereas `null`s are harder to handle accidentally.

(if these are strictly only for tests and wouldn't show up elsewhere, then it might not be worth it?)

Reply:
Yup, this is only for tests and should never show up anywhere else!